crates/bin/node/src/logic/inputs.rs

```diff
@@ -81,6 +84,22 @@ pub async fn process_melt_inputs<'a>(
             .await
             .map_err(Error::KeysetCache)?;
 
+        // Validate amount doesn't exceed max_order
+        let max_order = keyset_info.2;
```

Comment:
Is it possible to return the `CachedKeysetInfo` struct from keyset_info? rather than a tuple. To improve readability